tests/app/dao/test_users_dao.py

```diff
@@ -44,26 +44,6 @@ def test_name():
     return f'Test User {uuid.uuid4()}'
 
 
-@pytest.mark.serial  # Ensures only one user in the database
```

Comment:
Not sure why they thought it would add more than one, but this is an unnecessary test.